testgrid/tgrun/hack/os-spec.yaml

```diff
@@ -0,0 +1,71 @@
+- id: amzn-20
```

Comment:
did you mean to comment the two 'hack' files?

Reply:
Yeah, these are referenced [here](https://stackoverflow.com/c/replicated/questions/339) for testing, so I thought it pertinent to refresh them.